packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts

```diff
@@ -0,0 +1,73 @@
+import { Page } from '@playwright/test';
+import { BackendType, NetworkSymbol } from 'suite-common/wallet-config/src';
+
+import { waitForDataTestSelector } from '../common';
+
+class SettingsActions {
+    async toggleDebugModeInSettings(window: Page) {
+        const settingsHeader = window.getByTestId('@settings/menu/title');
+        await settingsHeader.waitFor({ state: 'visible', timeout: 10_000 });
+        const timesClickToSetDebugMode = 5;
+        for (let i = 0; i < timesClickToSetDebugMode; i++) {
+            // eslint-disable-next-line no-await-in-loop
+            await settingsHeader.click();
+        }
+        await window.getByTestId('@settings/menu/debug').waitFor({ state: 'visible' });
+    }
+
+    async goToDesiredSettingsPlace(
+        window: Page,
+        desiredLocation: 'debug' | 'general' | 'device' | 'wallet',
+    ) {
+        let desiredLocationWaiter: string;
```

Comment:
NIT: Why call it `...Waiter`? It seams to me better naming `desiredLocationDataTest`